File: codevs_samplecode/sample/Sample.java

import java.util.Scanner;

public class Sample {
	public static void main(String[] arg){
		Scanner scan  = new Scanner(System.in);
		String buf=scan.nextLine();
		int map,level;
		map = Integer.parseInt(buf);
		for(int i=0;i<map;i++){
			while(true){
				String tmp=scan.nextLine();
				if(tmp.equals("END")){
					level = Integer.parseInt(buf);
					break;
				}else{
					buf=tmp;
				}
			}
			for(int j=0;j<level;j++){
				while(true){
					buf=scan.nextLine();
					if(buf.equals("END")){
						System.out.println("0");
						break;
					}
				}
			}
		}
	}
}

File: codevs_samplecode/sample/sample.c

/**
 * CodeVS c sample
 */

#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main(){
	int map,level,i,j;
	char buf[1024];

	fgets(buf,1024,stdin);
	map=atoi(buf);
	for(i=0;i<map;i++){
		char tmp[1024];
		while(1){
			fgets(tmp,1024,stdin);
			if(strcmp(tmp,"END\n")==0){
				level=atoi(buf);
				break;
			}else{
				strcpy(buf,tmp);
			}
		}
		for(j=0;j<level;j++){
			while(1){
				fgets(buf,1024,stdin);
				if(strcmp(buf,"END\n")==0){
					puts("0");
					fflush(stdout);
					break;
				}
			}
		}
	}
	return 0;
}
